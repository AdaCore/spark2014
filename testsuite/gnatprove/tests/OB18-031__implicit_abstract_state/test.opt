ALL XFAIL implicit abstract state
