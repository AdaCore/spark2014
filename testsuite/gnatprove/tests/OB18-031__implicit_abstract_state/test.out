interfering_volatile_attribute.adb:7:20: info: initialization of "X" proved
interfering_volatile_attribute.ads:3:10: info: nontermination of task proved (CVC4: 1 VC)
